MSAL/src/native_auth/public/MSALNativeAuthChallengeTypes.h

```diff
@@ -25,13 +25,21 @@
 #ifndef MSALNativeAuthChallengeTypes_h
 #define MSALNativeAuthChallengeTypes_h
 
-/// The set of capabilities that an application wishes to support for Native Auth operations.
-///
-/// Valid options are:
-/// * OOB: The application can support asking a user to supply a verification code that is sent by email.
-/// * Password: The application can support asking a user to supply a password
+#import <Foundation/Foundation.h>
```

Comment:
Why do we have to add this import?

Reply:
The import is needed because MSALNativeAuthChallengeTypes is imported in `MSAL.h` with `#import <MSAL/MSALLegacySharedAccountsProvider.h>` 
If the import is not added here, it will fail creating the documentation when `sourcekitten` is run because `NS_OPTIONS` is not found

Reply:
Got it. Makes sense. Thanks.